### ISSUE TITLE
Introduction

### ISSUE THREAD

**Issue by robmarkcole** [debsahu/PiCamMovidius#1]
Hi
nice to see someone else interested in object detection in HA. I've a movidius too and was planning to experiment with making this available to HA via a rest API, so the movidius could be attached to any computer on my network. 
Any I will keep an eye on how this progresses. 
Cheers

**Comment by debsahu** [comment]
Hi Robin,

One can use flask and post jpeg frames to a webserver. See https://github.com/debsahu/Smart-Security-Camera . But what you are saying is you want to expose NCS API as RESTful API, that would be interesting. Let me know if you need help.

Deb.

**Comment by robmarkcole** [comment]
Yes it could be on another pi, and ideally would allow you to just add more movidius sticks as required
